python/foglamp/services/south/server.py

```diff
@@ -169,6 +171,31 @@ class Server(FoglampMicroservice):
                 _LOGGER.exception('Failed to poll for plugin {}, retry count: {}'.format(self._name, try_count))
                 await asyncio.sleep(2)
 
+    async def _register_interest_via_core(self, microservice_id, category):
+        """Register interest
+
+        Args:
+            microservice_id: microservice uuid from the service registry
+            category: name to get register interest with microservice id
+
+        Returns:
+               registration id and message
+        """
+        core_management_port = Parser.get('--port')
```

Comment:
We should not need to do any of this. We are derived from the FoglampMicroservice class, which is derived fro FogLAMP process, so we have an instance of a MicroserviceManagement class which includes a method to register interest.  We do not need to parse argument or build URL's this is all done for us. In fact we should not do this as it will fail in some cases when we have distributed services in the future.